<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage</class>
 <widget class="QWidget" name="SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="Vanilla::ToggleButton" name="horizonNavigation" native="true"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="settingStackedPage">
     <widget class="DefaultSettings" name="defaultSettings"/>
     <widget class="AccountsInfo" name="accountsInfo"/>
     <widget class="AdvanceSettings" name="advanceSettings"/>
     <widget class="PluginsInfo" name="pluginsInfo"/>
     <widget class="About" name="about"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Vanilla::ToggleButton</class>
   <extends>QWidget</extends>
   <header>VanillaStyle/Widgets/ToggleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>About</class>
   <extends>QWidget</extends>
   <header>ClientUi/Setting/About.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AccountsInfo</class>
   <extends>QWidget</extends>
   <header>ClientUi/Setting/AccountsInfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AdvanceSettings</class>
   <extends>QWidget</extends>
   <header>ClientUi/Setting/AdvanceSettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DefaultSettings</class>
   <extends>QWidget</extends>
   <header>ClientUi/Setting/DefaultSettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PluginsInfo</class>
   <extends>QWidget</extends>
   <header>ClientUi/Setting/PluginsInfo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
